backend/src/controllers/api.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response, Router } from "express";
+
+const userSchema = require("../models/UserSchema");
+const productSchema = require("../models/ProductSchema");
+
+const apiController: Router = Router();
+
+apiController.get("/", (req: Request, res: Response): void => {
+  res.send(`You requested GET /users`);
+});
+
+apiController.post("/updateUser", (req: Request, res: Response): void => {
+  res.send("Thanks for the update man <3");
+});
+
+apiController.post("/", (req: Request, res: Response): void => {
+  const user = new userSchema(
+    req.body.firstName,
+    req.body.lastName,
+    req.body.gronies
```

Comment:
I believe the correct spelling is groenies

Reply:
Are you sure?